packages/client/hmi-client/src/components/sidebar-panel/documents-sidebar-panel.vue

```diff
@@ -1,24 +1,18 @@
 <template>
-	<main>
-		<header>Documents space</header>
-		<section>
-			Project Documents:
-			<div class="document-list-container">
-				<div
-					v-for="doc in documents"
-					:key="doc._gddid"
-					class="doc-link"
-					:class="{ active: doc._gddid === docID }"
-					@click="openDocumentPage(doc)"
-				>
-					<span>{{ formatTitle(doc) }}</span>
-					<span class="doc-delete-btn" @click.stop="removeDocument(doc)">
-						<IconClose32 />
-					</span>
-				</div>
-			</div>
-		</section>
-	</main>
+	<div class="document-list-container">
```

Comment:
Why did you remove the side panel header text?

Reply:
I did not - this is build on top of @YohannParis 's sidebar changes.